ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -219,6 +350,20 @@ private void writeHeader( HttpServletResponse response )
         }
     }
 
+	public String getIpAddress()
```

Comment:
This is very likely to give invalid results (returning localhost, or network-local addresses). I've been battling with this before, and have tentatively come up with https://github.com/guusdk/external-ip-check - would that be useful here?